L1Trigger/L1TMuonBarrel/src/L1MuBMAssignmentUnit.cc

```diff
@@ -196,8 +196,8 @@ void L1MuBMAssignmentUnit::PhiAU(const edm::EventSetup& c) {
   phi_8 += sectordiff*48;
 
   int phi = phi_8 + 24;
-  if (phi >  60) phi =  60;
-  if (phi < -8) phi = -8;
+  if (phi >  69) phi =  69;//upper bound on phi assignment - phi is calculated in emulator all muons are<=60 but theoretically we can go up to 69
```

Comment:
@gkaratha 
I am sorry, but I don't understand this.  Please rephrase. 

Why was this value "60" before?  Was this a bug?
Where does the the number "69" come from ?